test/markdown/image.js

```diff
@@ -24,6 +25,28 @@ describe('Marpit image plugin', () => {
       .use(inlineSVG)
       .use(image)
       .use(backgroundImage)
+      .use(headerAndFooter)
+
+  describe('Regular image', () => {
+    const [token] = md().parseInline('![](https://example.com/image.jpg)')
+    const [imageToken] = token.children
+
+    it('uses primitive string as src attribute', () =>
+      expect(typeof imageToken.attrGet('src')).toBe('string'))
+
+    context('with header image via directive', () => {
+      const tokens = md().parse(
+        '<!-- header: "![](header.png)" -->\n\n![](content.png)'
```

Comment:
NOTE: Image in header directive is an edge case of #165 that try to override exists `State#process` function twice.